static-analysis/StaticAnalysisChapter.md

```diff
@@ -296,7 +295,7 @@ julia> e.args[2] # three lists of variable metadata
  {}                                       
 ~~~
 
-The first list above contains the names of all local variables; we only have one (`z`) here. The second list contains a tuple for each variable in and argument to the method; each tuple has the variable name, the variable's inferred type, and a number. The number conveys information about how the variable is used, in a machine (rather than human) friendly way. The last list is of captured variable names; it's empty in this example.
+The first list above contains the names of all local variables; we only have one (`z`) here. The second list contains a tuple for each variable in and argument to the method; each tuple has the variable name, the variable's inferred type, and a number. The number conveys information about how the variable is used, in a machine- (rather than human-) friendly way. The last list is of captured variable names; it's empty in this example.
```

Comment:
I used the parentheses as an aside. I think a sentence using the parens in the same way would be: "I have a brother (George) who's super tall."

Does that make sense?